src/lakefs_spec/client_helpers.py

```diff
@@ -63,3 +63,8 @@ def revert(client: LakeFSClient, repository: str, branch: str, parent_number: in
     client.branches_api.revert_branch(
         repository=repository, branch=branch, revert_creation=revert_creation
     )
+
+
+def rev_parse(client: LakeFSClient, repository: str, ref: str, parent: int = 0) -> str:
+    revisions = client.refs_api.log_commits(repository=repository, ref=ref).results
```

Comment:
Could we use the `after` and `limit` options here for `log_commits`? That optimizes bandwidth and amount of results (we only need at most `parent` results, if we have a merge then it might go to `2 * parent` (?)).

Just out of interest, have you checked what the `prefixes` and `objects` options do?

Reply:
`prefixes` allows to restrict the returned results to the commits that touched a certain directory. `objects` is the same for files.  I've added both to `rev_parse`. 

When specified, one can further the history to the commits which touched a certain directory or file.

Reply:
I implemented `limit` with `2*(parent +1)`. I don`t think there is a possibility to check if a merge happened without first fetching the log (which would drive the bandwidth saving ad absurdum). 

Did not use `after`. One would need to know the a commit SHA after which one wants to get the results. When you look up the commit sha, you don't need this helper function, right?